.circleci/config.yml

```diff
@@ -101,9 +101,9 @@ jobs:
 #      - restore_cache:
 #          keys:
 #            - judokit-reactnative-sample-ios-cocoapods-{{ checksum "sample-app/ios/Podfile" }}
-      - restore_cache:
-          keys:
-            - judokit-reactnative-sample-ios-ruby-gems-{{ checksum "sample-app/Gemfile.lock" }}
+      # - restore_cache:
+      #     keys:
+      #       - judokit-reactnative-sample-ios-ruby-gems-{{ checksum "sample-app/Gemfile.lock" }}
```

Comment:
Is this necessary?

Reply:
Honestly I am not sure, do we want it to pull down gems/pods/deps every time?